backend/db/ddl-12-csets_to_ignore.jinja.sql

```diff
@@ -1,6 +1,4 @@
 -- Table: csets_to_ignore ----------------------------------------------------------------------------------------------
-
-/*
 CREATE OR REPLACE VIEW {{schema}}csets_to_ignore{{optional_suffix}} AS
```

Comment:
@Sigfried The best option seemed to be to reactivate the DDL for `csets_to_ignore`. I couldn't delete the old version of `all_csets` because `csets_to_ignore` was dependent.

Another option is simply to delete that table. I know you told me not to worry about dealing with it last week, but I don't know if that is still the case, or if that also means I can delete it.